src/main/kotlin/edu/cornell/cs/apl/viaduct/protocols/ABY.kt

```diff
@@ -29,4 +32,21 @@ class ABY(hosts: Set<Host>) : MPCProtocol, SymmetricProtocol(hosts) {
 
     override fun hashCode(): Int =
         hosts.hashCode()
+
+    override fun compareTo(other: Protocol): Int {
```

Comment:
This should be implemented at a higher level, i.e., we should have a function for comparing two `Protocol`s.

Reply:
you already did this in a different PR